portal/server/routes/contract/canIssueOrEditIssueFacility.js

```diff
@@ -1,6 +1,4 @@
-const canIssueOrEditIssueFacility = (userRoles, deal, facility) => {
-  const isMaker = userRoles.includes('maker');
-
+const canIssueOrEditIssueFacility = (deal, facility) => {
```

Comment:
Out of curiosity, why did you make these changes? I'd be careful touching this as mine touches it heavily